### PR TITLE
Fixes Stimulants Speedbuff

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Fixes behavior with stimulants' speedbuff.

I hate how this has to be set twice.

### CODE REVIEW

**Comment by duncathan** [comment]
Holy fuck that is a shitty looking proc
Not your fault tho

**Comment by Fox-McCloud** [comment]
It's pretty bad, yeah. Basically the reagent itself sets the flag then it checks for this on the life loop...

File: code/modules/reagents/Chemistry-Holder.dm [hunk 1]
```diff
@@ -400,7 +400,7 @@ var/const/PATCH = 4 //patches
 
 /datum/reagents/proc/check_gofast(mob/M)
 	if(istype(M, /mob))
-		if(M.reagents.has_reagent("unholywater")||M.reagents.has_reagent("nuka_cola"))
+		if(M.reagents.has_reagent("unholywater")||M.reagents.has_reagent("nuka_cola")||M.reagents.has_reagent("stimulants"))
```
Comment (Miauw62) [inline code/modules/reagents/Chemistry-Holder.dm]: could you please make these three into a single for loop? has_reagent shows up pretty high in the profile iirc, and combining these three into a single for loop would probably make it more efficient.

@RemieRichards is this a good idea code quality-wise though?

Comment (Fox-McCloud) [inline code/modules/reagents/Chemistry-Holder.dm]: I'd rather saving the refactoring to someone else to be honest; I just wanted to get stimulants in a functioning status.

**Comment by Razharas** [comment]
Is this good to merge?

**Pull request merged**